firewood/src/db.rs

```diff
@@ -373,9 +376,9 @@ impl<S: ShaleStore<Node> + Send + Sync> DbRev<S> {
     }
 
     /// Verifies a range proof is valid for a set of keys.
-    pub fn verify_range_proof<N: AsRef<[u8]> + Send, K: AsRef<[u8]>, V: AsRef<[u8]>>(
+    pub fn verify_range_proof<NS: AsRef<[u8]> + Send, K: AsRef<[u8]>, V: AsRef<[u8]>>(
```

Comment:
Nit: I don't want to get too hung up on names, but why `NS`? Based on this PR, my brain is primed to think NS is short for "node source," which it doesn't appear to be in this case.

Reply:
Good point; I just quickly resolved the name collision and it needs a better letter.